geomstats/geometry/invariant_metric.py

```diff
@@ -778,21 +745,28 @@ def parallel_transport(
 
         def acceleration(state, time):
             """Compute the right-hand-side of the parallel transport eq."""
```

Comment:
Add parameters description 🙏

Reply:
I'll do it. (Notice this was fixed in this PR, not added now.)